hat/assets/js/apps/Iaso/domains/projects/config.tsx

```diff
@@ -16,56 +12,79 @@ import { FeatureFlag } from './types/featureFlag';
 import { Project } from './types/project';
 
 export const baseUrl = baseUrls.projects;
-export const columns = (
-    formatMessage: (msg: IntlMessage) => string,
+export const useColumns = (
     saveProject: (s: Project) => Promise<any>,
-): Array<Column> => [
-    {
-        Header: formatMessage(MESSAGES.projectName),
-        accessor: 'name',
-    },
-    {
-        Header: formatMessage(MESSAGES.appId),
-        accessor: 'app_id',
-    },
-    {
-        Header: formatMessage(MESSAGES.featureFlags),
-        accessor: 'feature_flags',
-        sortable: false,
-        Cell: settings =>
-            settings.value
-                .map(fF =>
-                    MESSAGES[fF.code.toLowerCase()]
-                        ? formatMessage(MESSAGES[fF.code.toLowerCase()])
-                        : fF.name || fF.id,
-                )
-                .join(', ') || textPlaceholder,
-    },
-    {
-        Header: formatMessage(MESSAGES.actions),
-        accessor: 'actions',
-        resizable: false,
-        sortable: false,
-        Cell: (settings): ReactElement => (
-            <Box
-                display="flex"
-                alignItems="center"
-                gap={1}
-                justifyContent="center"
-            >
-                <EditProjectDialog
-                    initialData={settings.row.original}
-                    saveProject={saveProject}
-                    dialogType="edit"
-                    iconProps={{}}
-                />
-                {settings.row.original.qr_code && (
-                    <QrCode qrCode={settings.row.original.qr_code} />
-                )}
-            </Box>
-        ),
-    },
-];
+): Array<Column> => {
+    const { formatMessage } = useSafeIntl();
+    return useMemo(() => {
+        return [
+            {
+                Header: formatMessage(MESSAGES.projectName),
+                accessor: 'name',
+                Cell: settings => {
+                    const textColor = Color(
+                        settings.row.original.color,
+                    ).isDark()
+                        ? '#fff'
+                        : '#000';
```

Comment:
Dunno if this would work:

```suggestion
                        ? 'white'
                        : 'black';
```